hack/operator-sdk-run.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -euo pipefail
+
+make operatorSDK
+
+file_path="$(pwd)/bundle/manifests/addon-operator.clusterserviceversion.yaml"
+
+if [ -e "$file_path" ]; then
+line=$(git diff -I'^    createdAt: ' | wc -l)
+  if [ "$line" -gt 0 ] ;then
```

Comment:
Better to use `! - z` or `-n` for string evalutations.

```suggestion
  if [[ -n $line ]]; then
```

Reply:
Why not omit the word count instead? A bit hacky IMO, `git diff` will be an empty string if `createdAt` isn't changed.